src/NBitcoin/Transaction.cs

```diff
@@ -1190,7 +1190,7 @@ public static Transaction Load(string hex, Network network, ProtocolVersion vers
                 throw new ArgumentNullException(nameof(network));
 
             Transaction transaction = network.Consensus.ConsensusFactory.CreateTransaction();
-            transaction.FromBytes(Encoders.Hex.DecodeData(hex), version, network);
+            transaction.FromBytes(Encoders.Hex.DecodeData(hex), version);
```

Comment:
Are you sure you don't need the network or consensusFaactory when you Load a transaction?

Reply:
@bokobza To put it another way, if we create the Transaction object outside of the Load method we need to pass in the network as we need to create it via the network's consensus factory. I.e. with the Clone method we call Load, which requires the network.

That said, I'm going to look at removing the clone method as its confusing as to which method to call to create\clone a transaction object.

Reply:
That's true, having more methods to do the same thing is very confusing.
Still, to clone a method, calling Load and having to serialize the object to bytes is not very intuitive.
For a next PR, how about these:
```
/// Creates a transaction by cloning the transaction passed as parameter.
public Transaction(Transaction transaction)
{}

/// Creates a transaction by cloning the transaction passed as parameter.
public Transaction(string hex, Network network)
```

So you could do `Transaction trx = new Transaction(oldTransaction);`

Reply:
@bokobza Absolutely! 👍